ai_agents/agents/ten_packages/extension/tencent_tts_python/extension.py

```
@@ -373,13 +373,13 @@ async def _process_audio_data(self) -> None:
-                self.current_request_id,
-                ModuleError(
+            await self.send_tts_error(
+                request_id=self.current_request_id,
+                error=ModuleError(
                     message=str(e),
                     module=ModuleType.TTS,
                     code=ModuleErrorCode.NON_FATAL_ERROR,
-                    vendor_info={},
+                    vendor_info=ModuleErrorVendorInfo(vendor=self.vendor()),
                 ),
             )
```

Pylint (4.0.0)

[refactor] Too many nested blocks (6/5)

(R1702)